src/Nethermind/Nethermind.JsonRpc/IJsonRpcConfig.cs

```diff
@@ -26,6 +26,12 @@ public interface IJsonRpcConfig : IConfig
         [ConfigItem(Description = "Host for JSON RPC calls. Ensure the firewall is configured when enabling JSON RPC. If it does not work with 117.0.0.1 try something like 10.0.0.4 or 192.168.0.1", DefaultValue = "\"127.0.0.1\"")]
         string Host { get; set; }
 
+        [ConfigItem(Description = "Defines whether the JSON RPC tracers will have timeout enabled. Default timeout is 5 seconds per tracer usage.", DefaultValue = "false")]
+        bool TracerTimeoutEnabled { get; set; }
```

Comment:
remove this and always enable tracer, just set the default timeout to something higher (20 seconds)